rtc_data_service/src/attestation/service.rs

```diff
@@ -0,0 +1,22 @@
+use actix::Addr;
+use actix_web::{error::ErrorInternalServerError, get, web, HttpRequest};
+
+use crate::auth_enclave_actor::AuthEnclaveActor;
+use crate::merge_error::*;
+
+use super::AttestationMessage;
+
+#[get("auth/attest")]
+pub async fn req_attestation_jwt(
+    _req: HttpRequest,
+    enclave: web::Data<Addr<AuthEnclaveActor>>,
+) -> actix_web::Result<String> {
+    let result = enclave
+        .send(AttestationMessage::default())
+        .await
+        .merge_err();
+    match result {
+        Ok(resp) => Ok(resp),
+        Err(err) => Err(ErrorInternalServerError(err)),
+    }
+}
```

Comment:
Hmm, we probably don't want to diverge our attestation endpoint implementations like this. Currently, the two endpoints are:

* `rtc_data_service::handlers::data_enclave_attestation`
* `rtc_data_service::attestation::service::req_attestation_jwt`

They're almost identical, but they're in different places, with some unnecessary implementation differences. We probably just want to settle on one interface and structure for both, and share common code where possible, rather than duplicating?